distribution/lib/Standard/Table/0.0.0-dev/src/Io/Excel.enso

```diff
@@ -0,0 +1,184 @@
+from Standard.Base import Integer, Text, Nothing, Boolean, Illegal_Argument_Error, Any, Error, Panic, File, Vector, False, IO
+from Standard.Base.Error.Problem_Behavior as Problem_Behavior_Module import Problem_Behavior
+
+import Standard.Table.Data.Table
+from Standard.Table.Error as Error_Module import Invalid_Location
+
+polyglot java import org.enso.table.format.xlsx.Range as Java_Range
+polyglot java import org.enso.table.format.xlsx.Reader
+
+polyglot java import java.lang.IllegalArgumentException
+polyglot java import java.io.IOException
+polyglot java import org.apache.poi.UnsupportedFileFormatException
+
+## Specified the part of an Excel Workbook to Read
+type Excel_Section
+    ## Gets a list of sheets within a workbook
+    type Sheet_Names
+
+    ## Gets a list of named ranges within a workbook
+    type Range_Names
+
+    ## Gets the data from a specific sheet. Column names are the Excel column names.
+    type Sheet (sheet:(Integer|Text)) (skip_rows:(Integer|Nothing)=Nothing) (row_limit:(Integer|Nothing)=Nothing)
+
+    ## Gets a specific range (taking either a defined name or external style address) from the workbook
+    type Range (address:(Text|Excel_Range)) (skip_rows:(Integer|Nothing)=Nothing) (row_limit:(Integer|Nothing)=Nothing)
+
+
+## Specifies a range within Excel and provides various ways of
+type Excel_Range
+    type Excel_Range java_range:Java_Range
```

Comment:
This sentence ends too early.

Also it should be moved/copied to the declaration below because unless I'm mistaken this is the comment that will be displayed in the IDE.